doc/userDocs/usersGuide.md

```diff
@@ -1551,6 +1550,23 @@ As a result, when using node labels it is recommended that you label all of your
 worker nodes with either `wvrole=driver` or `wvrole=sut`.  This will give you the 
 best control over pod placement.
 
+### Inter-pod affinity and anti-affinity<a name="pod-affinity"></a>
+
+With the Weathervane 2.1 release, configuration parameters were provided to manage the addition of pod affinity and anti-affinity rules with weights to the Weathervane pod manifests. 
+This provides more control for which nodes the pods are eligible to be scheduled on.
+
+| Parameter:                      | Description: |
+| ------------------------------- | ------------ |
+| `serviceTypeAntiAffinity`       | If true, a podAntiAffinity rule will be used to create anti-affinity among pods of the same type across all application instances. (Default: true) |
+| `serviceTypeAffinity`           | If true, podAffinity and podAntiAffinity rules will be used to create affinity among pods of the same type and anti-affinity among pods of different types.  These will apply across all application instances. (Default: false) |
+| `podInstanceAffinity`           | If true, a podAffinity rule will be used to create affinity among pods of the same application instance. (Default: false) |
+| `podInstanceAffinityWeight`     | The weight to apply to the podAffinity rule if podInstanceAffinity is true. (Default: 50) |
+| `serviceTypeAntiAffinityWeight` | The weight to apply to the podAntiAffinity rule if serviceTypeAntiAffinity is true. (Default: 100) |
+
+Note there are natural restrictions on the combination of these parameters on a given run. The serviceTypeAffinity rule can not be true when serviceTypeAntiAffinity or podInstanceAffinity are true.
+
```

Comment:
Why might a user be interested in setting serviceTypeAffinity, podInstanceAffinity? Why might they need to change the weight? Why were they added to the benchmark? Or likewise, should users not usually have to modify these policies?

Reply:
They are mostly for experimentation.  We used them to understand the performance impacts.
We had a least one external request for serviceTypeAffinity functionality.
Normal users won't typically modify these.

Reply:
The changes you made look good to me, thank you. It would be nice to have a sentence explaining to users why they would or wouldn't use these parameters, such as "These parameters will not need to be modified in typical use." But it's not a must have for me.

Reply:
Done